src/pages-and-resources/discussions/DiscussionsSettings.jsx

```diff
@@ -114,7 +122,18 @@ function DiscussionsSettings({ courseId, intl }) {
               eventKey={SELECTION_STEP}
               title={intl.formatMessage(messages.providerSelection)}
             >
-              <AppList />
+              {courseDetail
+                && !canChangeProviders
+                && (
+                  <Alert variant="warning">
+                    <FormattedMessage
```

Comment:
We are already using `messages.js` in this file, I think it makes sense to move this message to that file as well.

Reply:
I was under the impression that using it like this was the recommended way, as [described in the react-intl docs](https://formatjs.io/docs/getting-started/message-declaration#pre-declaring-using-definemessage-for-later-consumption-less-recommended).

I can change it though, just wanted to confirm first.

Reply:
Acknowledged that this should be changed. 👍

Reply:
I think this is still needed to be moved to message.js file

Reply:
Done!